### ISSUE TITLE
Release 4.9.1

### ISSUE THREAD

**Issue by blairconrad** [FakeItEasy/FakeItEasy#1454]
- [x] Address all coding and style violations detected during a *Release* build (send a regular pull request which must be merged before continuing)
- [x] ensure completeness of draft [release notes](https://github.com/FakeItEasy/FakeItEasy/releases), including non-owner contributors, if any (move release notes forward from any pre-releases to the current release)
- [x] run `tools\prepare_release.cmd x.y.z` to:
    - change "vNext" names to appropriate x.y.z version for the next GitHub Release, milestone, and release checklist (this issue)
    - create a new draft GitHub Release, milestone, and release checklist
    - create release/x.y.z branch on upstream, branching from develop
    - submit a pull request to merge the release/x.y.z branch to master
- [x] merge the pull request, _which may be done by the PR creator, unlike a regular pull request_
- [x] tag master with the new version number and push to upstream. This will cause the build server to
    - push the nupkgs to [nuget.org](https://www.nuget.org/profiles/FakeItEasy) and
    - attach all nupkgs from the tagged build to the GitHub Release.
- [x] wait for NuGet package validation
- [x] de-list pre-release NuGet packages if present
- [x] publish the GitHub release
- [x] tweet, mentioning contributors, and post link as comment here for easy retweeting ;-)
- [x] post tweet in [Gitter](https://gitter.im/FakeItEasy/FakeItEasy)
- [x] post a link to the GitHub Release in each issue in this milestone, with thanks to contributors
- [ ] close this issue
- [ ] close the [milestone](https://github.com/FakeItEasy/FakeItEasy/milestones?state=open) for this release

**Comment by blairconrad** [comment]
I can push out 4.9.1 once #1460 is fixed, if you like.

**Comment by thomaslevesque** [comment]
> I can push out 4.9.1 once #1460 is fixed, if you like.

Sure. Or I can do it, if one release this week was enough for you :wink:

**Comment by blairconrad** [comment]
```
Get FakeItEasy 4.9.1!
Fixed: Faked method calls from within concurrent threads don't get recognized by FakeItEasy all of the time

Thanks to Toni Riegler for reporting the bug, and to @adamralph for continuing to upgrade our build tooling

https://github.com/FakeItEasy/FakeItEasy/releases/4.9.1
```

```
This change has been released as part of [FakeItEasy 4.9.1](https://github.com/FakeItEasy/FakeItEasy/releases/4.9.1).
```

**Comment by blairconrad** [comment]
https://twitter.com/fakeiteasyfx/status/1045641213372706816

This change has been released as part of [FakeItEasy 4.9.1](https://github.com/FakeItEasy/FakeItEasy/releases/4.9.1).
